rpc/tendermint/test/common.go

```diff
@@ -32,7 +32,7 @@ func TestWrapper(runner func() int) int {
 func DebugMain() {
 	t := &testing.T{}
 	TestWrapper(func() int {
-		testBroadcastTx(t, "HTTP")
```

Comment:
"Non-trivial delta out of context"... I'm just going to leave this [here](http://www.plainenglish.co.uk/)